Content.Server/GameObjects/Components/Atmos/GasAnalyzerComponent.cs

```diff
@@ -19,14 +20,15 @@ public class GasAnalyzerComponent : Component, IAfterInteract
     {
 #pragma warning disable 649
         [Dependency] private IServerNotifyManager _notifyManager = default!;
+        [Dependency] private ILocalizationManager _loc = default!;
```

Comment:
You should be able to use plain `Loc.GetString` (a static convenience wrapper), which will save you this dependency.